package.json

```
@@ -145,7 +145,7 @@
     "opossum": "^5.0.1",
     "promise-retry": "^2.0.1",
     "proxyquire": "^2.1.0",
-    "puppeteer-core": "^5.2.1",
+    "puppeteer-core": "^3.1.0",
```

If we want to make the version static, consider removing the `^` caret.

```suggestion
    "puppeteer-core": "3.1.0",
```

---

thanks! should remove in fact since each version of puppeteer is tied to a specific version of chromium; I've edited accordingly